doc/lsst.meas.algorithms/reference-catalog-schema.rst

```diff
@@ -30,21 +31,22 @@ Catalog Schema
 The following are optional; fields should only be present if the information is available for at least some objects.
     Numeric values are `nan` if unknown:
 
+    - ``coord_ra_coord_dec_Cov``: covariance between ``coord_ra`` and ``coord_dec`` (rad^2)
     - ``epoch``: date of observation as TAI MJD (day)
     - ``pm_ra``: proper motion along the direction of right ascension (rad/year, an Angle) = dra/dt * cos(dec)
     - ``pm_dec``: proper motion along the direction of declination (rad/year, and Angle)
     - ``pm_raErr``: uncertainty in `pm_ra` (rad/year)
     - ``pm_decErr``: uncertainty in `pm_dec` (rad/year)
-    - ``pm_ra_dec_Cov``: covariance between pm_ra and pm_dec (rad2/year2)
+    - ``pm_ra_pm_dec_Cov``: covariance between pm_ra and pm_dec (rad^2/year^2)
     - ``pm_flag``: set if proper motion, error or covariance is bad
     - ``parallax``: parallax (rad, an Angle)
     - ``parallaxErr``: uncertainty in `parallax` (rad)
     - ``parallax_flag``: set if parallax value or parallaxErr is bad
-    - ``coord_ra_pm_ra_Cov``: covariance between coord_ra and pm_ra (rad2/year)
-    - ``coord_ra_pm_dec_Cov``: covariance between coord_ra and pm_dec (rad2/year)
-    - ``coord_ra_parallax_Cov``: covariance between coord_ra and parallax (rad2/year)
-    - ``coord_dec_pm_ra_Cov``: covariance between coord_dec and pm_ra (rad2/year)
-    - ``coord_dec_pm_dec_Cov``: covariance between coord_dec and pm_dec (rad2/year)
-    - ``coord_dec_parallax_Cov``: covariance between coord_dec and parallax (rad2/year)
-    - ``pm_ra_parallax_Cov``: covariance between pm_ra and parallax (rad2/year)
-    - ``pm_dec_parallax_Cov``: covariance between pm_dec and parallax (rad2/year)
+    - ``coord_ra_pm_ra_Cov``: covariance between coord_ra and pm_ra (rad^2/year)
+    - ``coord_ra_pm_dec_Cov``: covariance between coord_ra and pm_dec (rad^2/year)
+    - ``coord_ra_parallax_Cov``: covariance between coord_ra and parallax (rad^2/year)
+    - ``coord_dec_pm_ra_Cov``: covariance between coord_dec and pm_ra (rad^2/year)
+    - ``coord_dec_pm_dec_Cov``: covariance between coord_dec and pm_dec (rad^2/year)
+    - ``coord_dec_parallax_Cov``: covariance between coord_dec and parallax (rad^2/year)
+    - ``pm_ra_parallax_Cov``: covariance between pm_ra and parallax (rad^2/year)
+    - ``pm_dec_parallax_Cov``: covariance between pm_dec and parallax (rad^2/year)
```

Comment:
Are we sure the units documented above are all correct? I guess the only one that's different is `pm_ra_pm_dec_Cov` (rad^2/year^2), which we did get right above?

Reply:
I realized that some of the old units were wrong -- `coord_ra_parallax_Cov` should be rad^2, not rad^2/year -- so I've changed this.